lib/rucio/client/metaconventionsclient.py

```diff
@@ -20,18 +20,20 @@
 
 from rucio.client.baseclient import BaseClient
 from rucio.client.baseclient import choice
-from rucio.common.utils import build_url
-from rucio.db.sqla.constants import KeyType
+from rucio.common.utils import build_url, is_client
 from typing import Union, Optional
 
+if not is_client:
+    from rucio.db.sqla.constants import KeyType
```

Comment:
Thank you for your quick reaction. Do you wish to consider using `typing.TYPE_CHECKING` instead? This is my own personal opinion, so I am open to keeping it as it is, but `typing.TYPE_CHECKING` feels more natural because it’s a conditional import that is used exclusively for type checking (and if you’re type checking, how likely is it that you don’t have the entire source?)

Reply:
I won't protest to that suggestion :).`TYPE_CHECKING` seems to be the more common solution anyways.